lms/djangoapps/ccx/views.py

```diff
@@ -237,14 +237,24 @@ def create_ccx(request, course, ccx=None):
     # Save display name explicitly
     override_field_for_ccx(ccx, course, 'display_name', name)
 
-    # Hide anything that can show up in the schedule
+    is_course_licensing_enable = configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
+    # if license enforcement is enabled and user is allowed to create ccx all units will be showed in schedule.
+    hide_units = not (
```

Comment:
You are not working only with units but with sections and subsections as well.
```suggestion
    hide = not (
```